src/Rings/FunctionField.jl

```diff
@@ -85,10 +85,23 @@ function (F::Nemo.fpField)(x::AbstractAlgebra.Generic.Frac{T}) where T <: Union{
   F(cst_num) // F(cst_denom)
 end
 
+function (F::Nemo.FqField)(x::AbstractAlgebra.Generic.Frac{T}) where T <: Union{FqPolyRingElem, FqMPolyRingElem}
+  num = numerator(x)
+  cst_num = constant_coefficient(num)
+  denom = denominator(x)
+  cst_denom = constant_coefficient(denom)
+  if (num != cst_num || denom != cst_denom) throw(InexactError(:fpFieldElem, fpFieldElem, x)) end
```

Comment:
```suggestion
  if (num != cst_num || denom != cst_denom) throw(InexactError(:FqFieldElem, FqFieldElem, x)) end
```
I think this should be changed, as there is no `fpFieldElem` anywhere in this function